README.md

```diff
@@ -155,6 +155,14 @@ Install the library using the `[dev]` option, as above.
   sphinx-build . ../build
   ```
 
+  If the example notebooks for the docs need to be updated, the corresponding 
+  python scripts in the `examples` directory should also be regenerated via:
+  ```python
+  cd docs/source/examples
+  python convert_notebooks.py
+  ```
+  Note that this will require the installation of `jupyter` and `nbconvert`.
+
```

Comment:
potentially point them to the installation `.[option]` that installs `jupyter` and `nbconvert` with everything, since I think we have one that does this